### PR TITLE
AADHomeRealmDiscoveryPolicy

### PR DESCRIPTION
**Pull request by PiyushDub**
#### Pull Request (PR) description
AADHomeRealmDiscoveryPolicy Initial Release

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADHomeRealmDiscoveryPolicy "AADHomeRealmDiscoveryPolicy-displayName-value"
+        {
+            ApplicationId         = $ConfigurationData.NonNodeData.ApplicationId;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1]: This should only be $ApplicationId and $CertificateThumbprint

File: Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    AccelerateToFederatedDomain         = $False
+                    AlternateIdLogin = MSFT_AADHomeRealDiscoveryPolicyDefinitionAlternateIdLogin {
+                        Enabled = $True
+                    }
+                }
+            );
+            DisplayName           = "displayName-value";
+            Ensure                = "Present";
+            IsOrganizationDefault = $False;
+            TenantId              = $OrganizationName;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1]: This should be TenantId

File: Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/2-Update.ps1 [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+    node localhost
+    {
+        AADHomeRealmDiscoveryPolicy "AADHomeRealmDiscoveryPolicy-displayName-value"
+        {
+            ApplicationId         = $ConfigurationData.NonNodeData.ApplicationId;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/2-Update.ps1]: Same, we need to fix the 3 authentication params

File: Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/3-Remove.ps1 [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                    AccelerateToFederatedDomain         = $False
+                    AlternateIdLogin = MSFT_AADHomeRealDiscoveryPolicyDefinitionAlternateIdLogin {
+                        Enabled = $True
+                    }
+                }
+            );
+            DisplayName           = "displayName-value";
+            Ensure                = "Absent";
+            IsOrganizationDefault = $False;
+            TenantId              = $OrganizationName;
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/3-Remove.ps1]: Same, we need to fix the 3 authentication params

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
Changes requested for the examples

File: Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1 (hunk 1 above)
Comment (PiyushDub) [reply Modules/Microsoft365DSC/Examples/Resources/AADHomeRealmDiscoveryPolicy/1-Create.ps1]: updating this and all below

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**